balancer-labs/frontend-v2
# Fix issues with APR Tooltip on Polygon

# Description

Fixes a few issue with Tooltips on Polygon / L2's since the General Boosted Pools update merged yesterday

- When BAL rewards min/max are the same, show them as one line item instead of having a breakdown.
- If a pool has reward tokens for staking, show them as a line item under staking rewards
- If BAL rewards min/max are the same and the pool has stakign rewards, show two line items of BAL, then rewards.
- Label stMATIC staking rewards

## Type of change


## How should this be tested?

- View the APR Tooltips on Polygon, ensure they look correct, aren't showing ranges where min/max are the same, and data isn't missing

## Visual context

Before: 

![2022-12-03-175330_208x244_scrot](https://user-images.githubusercontent.com/525534/205430896-6377e1c8-c332-4cd7-91b6-ab7d1cb0b582.png)

After: 

![2022-12-03-175304_262x212_scrot](https://user-images.githubusercontent.com/525534/205430898-d01e3cc2-d775-4014-aab3-6db4fe3fde73.png)

Without any additional staking reward tokens (just BAL):

![2022-12-03-180324_177x152_scrot](https://user-images.githubusercontent.com/525534/205431181-cd2f853e-5810-45a1-8a36-4ff9bf36018e.png)


## Checklist:
